android/KMEA/build.sh

```diff
@@ -7,12 +7,13 @@
 # KMW  - Keyman Web
 
 display_usage ( ) {
-    echo "build.sh [-no-kmw-build] | [-no-kmw] [-no-daemon]"
+    echo "build.sh [-no-kmw-build] | [-no-kmw] [-no-daemon] | [-no-test]"
     echo
     echo "Build Keyman Engine Android (KMEA) using Keyman Web (KMW) artifacts"
     echo "  -no-kmw-build           Don't build KMW. Just copy existing artifacts"
     echo "  -no-kmw                 Don't build KMW. Don't copy artifacts"
     echo "  -no-daemon              Don't start the Gradle daemon. Use for CI"
+    echo "  -no-test                Don't run the unit-test suite."
```

Comment:
btw, I believe **release** builds always run tests, There, the gradle task runs `build` which depends on `check` which depends on `test`.

Per gradle plugin [documentation](https://docs.gradle.org/current/userguide/java_plugin.html)

Reply:
So... this should be noted as a `-debug`-only option?  Or am I interpreting this wrong?